app/konnect-platform/cmek.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Customer-Managed Encryption Keys (CMEK)
+content_type: reference
+layout: reference
+description: 'Use Customer-Managed Encryption Keys (CMEK) in {{site.konnect_short_name}} to encrypt sensitive data using keys from your AWS Key Management Service (KMS) account.'
+breadcrumbs:
+  - /konnect/
+
+
+products:
+    - konnect-platform
+tags:
+  - encryption
+  - security
+  - compliance
+works_on:
+  - konnect
+faqs:
```

Comment:
Another FAQ that we can add is - can I bring in symmetric keys from other key manaegement services? The answer is no. CMEK only suports AWS KMS for now.